themes/default/partials/sidebar.handlebars

```diff
@@ -16,14 +16,14 @@
             {{#globals.classes}}
                 <li>
                     <a class="class" href="{{project.base}}/classes/{{name}}.html">{{namespace}}</a>
-                    <a href="{{project.base}}/modules/{{module}}.html" class="api-list-item-module">@{{module}}</a>
+                    <a href="{{../base}}/modules/{{module}}.html" class="api-list-item-module">@{{module}}</a>
```

Comment:
This is the usage, and the syntax sugar here is for variable access in out of scope, see: http://stackoverflow.com/questions/13645084/access-a-variable-outside-the-scope-of-a-handlebars-js-each-loop